lib/Telegram/Bot/Object/Message.pm

```diff
@@ -147,7 +147,9 @@ sent.
 sub reply {
   my $self = shift;
   my $text = shift;
-  return $self->_brain->sendMessage({chat_id => $self->chat->id, text => $text});
+  my $args = shift // {};
+
+  return $self->_brain->sendMessage({chat_id => $self->chat->id, text => $text, $args->%* });
```

Comment:
The postfix deref was introduced in Perl 5.20 as an experimental feature. We are requiring Mojolicious of at least version 8. Mojo introduces a lot of features by default and requires quite a new Perl, but 8 is actually quite old and only requires Perl 5.10.1. So this would break on the minimum requirement, and lots of combinations after that. The newest Mojo 9.34 only requires Perl 5.16.0, which is also not new enough to have this feature. I suspect it works for you because you have a very new Perl.

I suggest to stick with the old-fashioned dereferencing and do `%$args` here to maintain backward compatibility. Otherwise you'd have to update the dist.ini, but that will likely break people's production stuff.

Reply:
Agreed, it's not important enough to break backcompat that severely.